### PR TITLE
Update guest-access-PowerShell.md

### PR DESCRIPTION
**Pull request by Reeced40**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/1897
Changed wording

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/guest-access-PowerShell.md [hunk 1]
```diff
@@ -31,8 +31,8 @@ For details, see the section "Use PowerShell to control guest access" on the Man
   
 You can also use PowerShell to allow or block a guest user based on their domain. For example, let's say your business (Contoso) has a partnership with another business (Fabrikam). You can add Fabrikam to your Allow list so your users can add those guests to their groups. For more information, see [Allow/Block guest access to Office 365 groups](https://go.microsoft.com/fwlink/?linkid=854001).
   
-If you want to block guests in teams and still allow guests to access SharePoint sites, you can use Azure Active Directory Powershell cmdlets to disable the AllowGuestAccessToGroups parameter on the Company object, assuming external sharing is turned on for SharePoint sites.   
+If you want to block guests in teams and still allow guests to access SharePoint sites, you can use Azure Active Directory Powershell cmdlets to disable the AllowGuestToAccessGroups parameter on the Company object, assuming external sharing is turned on for SharePoint sites.
```
Comment (JohanFreelancer9) [inline Teams/guest-access-PowerShell.md]:
```suggestion
If you want to block guests in Teams and still want to allow them to access SharePoint sites, you can use Azure Active Directory Powershell cmdlets to disable the AllowGuestToAccessGroups parameter on the Company object, assuming external sharing is turned on for SharePoint sites.   
```

**Comment by LolaJacobsen** [comment]
@somakbhattacharyya - OK to merge?

**Pull request merged**